src/views/Workflow.vue

```diff
@@ -175,9 +201,57 @@ export default {
             }
           })
       }
+      if (this.deltaSubscriptions.length === 0) {
+        const vm = this
+        this.$workflowService
+          .startDeltasSubscription(WORKFLOW_TREE_DELTAS_SUBSCRIPTION, this.variables, {
+            next: function next (response) {
+              applyDeltas(response.data.deltas, vm.tree)
+              vm.isLoading = false
+            },
+            error: function error (err) {
+              vm.setAlert(new Alert(err.message, null, 'error'))
+              vm.isLoading = false
+            }
+          })
+      }
```

Comment:
The code above appears to be creating a new delta subscription for the tree. Below is the code for the new delta subscription for the table. This is one of the bugs I mentioned in our Element chat today.

We already have a delta subscription for the tree. So no need to create it again.